src/components/markdown/examples/markdown-composite.scss

```
@@ -0,0 +1,19 @@
+:host(limel-example-markdown-composite) {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+
+    limel-input-field {
+        height: 10rem;
+    }
+
+    fieldset {
+        padding: 0.5rem 0.5rem 3rem 0.5rem;
```

```suggestion
            padding-block: 1rem;
            padding-inline: 1rem;
            margin-inline: 0;
```

Since we are overriding the native styles and they are using the logical properties, it'd be nice to also override them with logical properties. :)